app/search/orderers/search_orderer.rb

```diff
@@ -0,0 +1,60 @@
+# This service is responsible for applying the correct ordering to a search
```

Comment:
(nit-pick) Shorten top-level docstrings by starting directly with what it does, e.g. `Applies...`. This applies to many files of this PR, e.g. the `lecture_media_orderer.rb` which starts with `This orderer...`.